examples/requirements.txt

```diff
@@ -1,4 +1,5 @@
-promptflow[azure]
+--extra-index-url https://azuremlsdktestpypi.azureedge.net/promptflow/
```

Comment:
merge blocker/reminder: manually bump versions in pyproject.toml to 1.10.0.dev0

Reply:
Updated.